public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataPane.tsx

```diff
@@ -175,6 +175,13 @@ function getStyles(theme: GrafanaTheme2) {
       borderLeft: 'none',
       borderBottom: 'none',
       borderTopRightRadius: theme.shape.radius.default,
+      flexGrow: 1,
+    }),
+    tabsBar: css({
+      flexShrink: 0,
+    }),
+    scroll: css({
+      background: theme.colors.background.primary,
     }),
```

Comment:
Why is this needed?

Reply:
I tested without it and could not see any change, there TabContent has a background set

Reply:
With:

https://github.com/grafana/grafana/assets/45561153/5862943a-3b40-4912-9304-22d3cb6d7c65

Without:


https://github.com/grafana/grafana/assets/45561153/a234feb4-2f74-494f-8d56-5e6f912ade82